src/components/templates/PartyMap/components/RoomAttendance/RoomAttendance.module.scss

```
@@ -0,0 +1,7 @@
+@use "scss/attendee/space";
+
+.RoomAttendance {
```

More visible it is no longer the old BEM.

```suggestion
.roomAttendance {
```